avidemux/common/ADM_commonUI/DIA_prefs.cpp

```diff
@@ -354,7 +360,20 @@ std::string currentSdlDriver=getSdlDriverName();
         framePriority.swallow(&menuEncodePriority);
         //framePriority.swallow(&menuIndexPriority);
         framePriority.swallow(&menuPlaybackPriority);
-
+#else
+        diaMenuEntry priorityEntries[] = {
+                     {2,      QT_TRANSLATE_NOOP("adm","Normal"),NULL}
+                    ,{3,      QT_TRANSLATE_NOOP("adm","Below normal"),NULL}
+                    ,{4,      QT_TRANSLATE_NOOP("adm","Low"),NULL}
+        };
+        diaElemMenu menuEncodePriority(&encodeNixPriority, QT_TRANSLATE_NOOP("adm","_Encoding priority:"), NB_ITEMS(priorityEntries), priorityEntries);
+        diaElemReadOnlyText encodePriorityText(NULL,
+                QT_TRANSLATE_NOOP("adm","After an encoding is finished, the restoration of the original priority is only possible with the proper privileges."),NULL);
```

Comment:
I would prefer a very clear hint that this is practically a one-way street affecting everything if I am not mistaken, not just encoding. If you agree, I'll change the text accordingly when merged.

Reply:
Ok.